README.md

```diff
@@ -265,8 +265,8 @@ holding the desired options for this instance. The possible options are:
   documented under the [get()](#get) method, along with an example. If
   specified, this function will be called each time a value is promoted from the
   L2 cache into the L1 (worker Lua VM). This function can perform arbitrary
-  serialization of the cached item to transform it into any Lua object _before_
-  storing it into the L1 cache. It can thus avoid your application from
+  de-serialization of the cached item to transform it into any Lua object
```

Comment:
This function is called `l1_serializer`, not `l1_deserializer`. Its purpose is to _serialize for L1 caching_. This change brings confusion.